src/Stryker.Core/Stryker.Core/Initialisation/InputFileResolver.cs

```diff
@@ -245,7 +245,7 @@ private void AppendExampleIfPossible(StringBuilder builder, IEnumerable<string>
             }
 
             builder.AppendLine("");
-            builder.AppendLine($"Example: --project-file={otherProjectReference}");
+            builder.AppendLine($"Example: --project={otherProjectReference}");
```

Comment:
```suggestion
            builder.AppendLine(otherProjectReference);
```

Reply:
@corentinaltepe Sorry if I was unclear, I meant that his whole method can be removed as I think it's more confusing than helpful to choose one project reference at random to give as a suggestion.

Reply:
I understand the intent. A commit was pushed removing the method, its calls and the corresponding assertion in one of the unit tests.

Slightly unrelated: the region `string helper methods` was removed because it contained only 1 method. And the method `BuildReferenceChoice` was made static as suggested by the IDE.